manifests/Manlaan/DistanceRings/1.0.3.json

```diff
@@ -0,0 +1,19 @@
+{
+    "manifest_version": "1",
+    "name": "Distance Rings",
+    "namespace": "DistanceRings",

```

Comment:
Make sure to use a nickname of yours as a namespace prefix for your modules.

Eg.  manlaan.Guild_Missions

1. The prefix should be the same as the name of your module folder where the manifest lies in this repository.
2. Don't just edit the namespace in this manifest. Also add the prefix to files and project of the module itself.

This makes sure that there won't be any conflicts with other modules.